requirements/REQUIREMENTS.md

```diff
@@ -0,0 +1,22 @@
+## Types of project users:
+* Any Internet user who is interested in looking at some statistics
+* Users who need data for various studies
+* Medical workers who can upload their data
+* Moderators who will work on the the product and upload new data + make sure that other users do not violate website policy/rules
+* An employee of a medical institution: an ordinary user + can upload data (they have the rights to do this, somehow the moderator will confirm this after passing the check) 
+
+## Use case diagram
+
+![](SEhw.png)
```

Comment:
Why does one use case bubble has arrow to another one without any label? What type of connection is used there?

Reply:
Fixed diagram